### DISCUSSION TITLE
[A-L] (2023/24) Foglio 6 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#146] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-09 172738](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/a7b59f94-525e-4b69-8493-795835362f99)

**Comment by cornflexxx** [comment]
Grazie al teorema di Lagrange sappiamo che i sottogruppi di $S_3$ si limitano a sottogruppi che hanno ordine divisore di $3!$:

1. $S_3$ stesso (ovviamente normale)
2. Il sottogruppo banale costituito dalla permutazione identità ( $\sigma\circ id\circ\sigma^{-1}=id\in Id_3\to Normale)$
---------------
3. Consideriamo ora il sottoinsieme delle permutazioni che spostano $a\to a+1$  o $a\to a+2$ quindi questo è formato da:


$$\phi =\begin{pmatrix}
1 & 2 & 3 \\
 2& 3 & 1
\end{pmatrix}$$

$$Id =\begin{pmatrix}
1 & 2 & 3 \\
 1& 2 & 3
\end{pmatrix}$$

$$\tau= \begin{pmatrix}
1 & 2 & 3 \\
 3& 1 & 2
\end{pmatrix}$$

Graficamente si può rappresentare come le rotazioni di un triangolo di 0°(o 360°),120° e 240°. Ho fatto varie prove su carta delle composizioni dei vari $\tau,\phi$  ed effettivamente rispetta questo comportamento, ad esempio la composizione $\tau\circ\tau=\phi$ , oppure $\tau=\phi^{-1}$ e viceversa. Con la rappresentazione grafica risulta abbastanza intuitivo che se ad esempio riflettiamo un triangolo rispetto alla bisettrice di uno degli angoli , la ruotiamo di un angolo qualunque (tra 120° e 240°) e infine lo riflettiamo di nuovo rispetto alla diagonale è chiaro che ci troviamo ancora all'interno del sottogruppo delle rotazioni, quindi $R_3$ è normale rispetto ad $S_3$ (ho fatto una verifica anche caso per caso). 

-----------------
4. Infine abbiamo l'ultima operazione che possiamo compiere su un triangolo ovvero quella di riflessione, già citata nel punto precedente. Si tratta del sottoinsieme di $S_3$ formato dalle trasposizioni (ovvero lascio fisso uno dei tre punti e scambio gli altri due). Non possiamo creare  un sottogruppo di ordine 4 comprendente tutte le trasposizioni più l'identità per due motivi:

-  un sottogruppo di ordine 4 non rispetterebbe il _th. di Lagrange_ in quanto: $4\nmid o(S_3)$
-  un sottogruppo formato dalle 3 trasposizioni più l'identità non è chiuso rispetto all'operazione di composizione e quindi non è effettivamente un gruppo.

Per risolvere questi due problemi consideriamo i 3 sottogruppi formati dalle singole trasposizioni e l'identità, effettivamente questi sottoinsiemi di $S_3$ formano dei sottogruppi rispetto alla composizione , oltre ad essere di ordine due. Possiamo verificare inoltre: 

$$\sigma=\begin{pmatrix}
1 & 2 & 3 \\
 1& 3 & 2
\end{pmatrix}$$

$\phi \circ \sigma \circ \phi^{-1}\notin Trasposizioni_{23}$. Riconducendoci ad un triangolo come nel caso precedente risulta più facile vedere che se ruotiamo di 240° , riflettiamo sulla bisettrice del vertice 1 (che in quel momenti si trova sull' originario vertice 3) ed infine ruotiamo di altri 120° non ci troveremo all'interno di $Trasposizioni_{23}$ , dopo aver riflesso la rotazione non avviene più in senso orario, bensì in senso antiorario , quindi non ritorniamo alla posizione di partenza. Allo stesso modo se imponiamo una prima riflessione rispetto ad un vertice, poi una seconda rispetto ad un vertice diverso dal primo ed infine una riflessione uguale alla prima, alteriamo in questo modo completamente la figura iniziale. Per essere ricondotta ad una trasposizione appartenente a $Trasposizioni_{23}$, dobbiamo comporre con una riflessione rispetto al vertice non ancora toccato dei 3 , oppure seguire di nuovo i passi fatti in precedenza.



-------------

In totale abbiamo 6 sottogruppi di cui 3 normali su $S_3$,  non ne esistono altri perchè possiamo escludere a priori quelli di ordine 5 o superiore di 6.

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/97389762/cf5c81fd-125b-4def-a686-7708695770be)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/1631b8aa-21d1-4649-b8bc-13a3b61ea55a)

**Comment by Elia-Belli** [comment]
**(1)** Il gruppo simmetrico $S_3$ contiene tutte le permutazioni di $\\{1,2,3\\}$, elenchiamole in forma di cicli:

$$S_3=\\{(1),(1\ 2),(1\ 3),(2\ 3),(1\ 2\ 3),(3\ 2\ 1)\\}$$

Abbiamo i sottogruppi banali $\\{(1)\\}$ l'identità e $S_3$ il gruppo stesso.
Per il Teorema di Lagrange i restanti sottogruppi hanno ordine $2$ o $3$: quelli di ordine $2$ sono generati dai $2-cicli$ mentre quello di ordine $3$ è generato da entrambi i $3-cicli$ ed è il sottogruppo alterno.

$$<(1\ 2)> =\\{(1),(1\ 2)\\}\ ;\ <(1\ 3)> =\\{(1),(1\ 3)\\}\ ;\ <(2\ 3)> =\\{(1),(2\ 3)\\}$$

$$<(1\ 2\ 3)> =\\{(1),(1\ 2\ 3),(3\ 2\ 1)\\}=A_3$$

**(2)** In $S_n$ , ad eccezione di $n=4$, sappiamo che gli unici sottogruppi normali sono quelli banali e quello alterno, nel nostro caso: $$\\{(1)\\}\ ;\ S_3\ ;\ A_3$$
